src/providers/local/user-pools.provider.ts

```diff
@@ -0,0 +1,121 @@
+import { computed, InjectionKey, provide, reactive, ref, watch } from 'vue';
```

Comment:
You can delete this lines but `InjectionKey` type (Note that types also will be auto-imported in the next version of unplugin autoimport that will be released this week 🎉).